app/lib/brexit_checker/account_jwt.rb

```diff
@@ -25,7 +25,12 @@ def scopes
   end
 
   def attributes
-    { transition_checker_state: criteria_keys }
+    {
```

Comment:
Do we have any tests around the saving of attributes?  It feels like something important that should have test coverage.

Reply:
I don't think we do, I'll add some